sys/net/vnet.c

```diff
@@ -248,8 +248,11 @@ vnet_alloc(void)
 	 * All use of vnet-specific data will immediately subtract VNET_START
```

Comment:
This comment should probably be rewritten to be more like the DPCPU one you've added.

Reply:
Done, though I've adjusted both comments a bit.